client/src/components/OptionsDrawer/OptionsDrawer.tsx

```diff
@@ -1,23 +1,39 @@
-import Typography from '@material-ui/core/Typography';
-import ListItem from '@material-ui/core/ListItem';
-import Divider from '@material-ui/core/Divider';
-import List from '@material-ui/core/List';
-import Drawer from '@material-ui/core/Drawer';
-import React from 'react';
-import ListItemText from '@material-ui/core/ListItemText';
-import ListItemIcon from '@material-ui/core/ListItemIcon';
+import { useState, useEffect, Dispatch, SetStateAction } from 'react';
+import { Typography, ListItem, ListItemText, ListItemIcon, List, Divider, Drawer } from '@material-ui/core';
 import SwapHorizontalCircleOutlinedIcon from '@material-ui/icons/SwapHorizontalCircleOutlined';
 import useStyles from './optionsDrawerStyles';
+import { getUserBoards, getBoard } from '../../helpers/';
+import { useKanban } from '../../context/useKanbanContext';
 import { IBoard } from '../../interface/Board';
 
 interface Props {
   open: boolean;
-  boards: Array<IBoard>;
```

Comment:
Not used any more (boards will be fetched in the drawer component rather than as a prop)